### PR TITLE
Added PR rules to the explictly outline the responsibilites.

### PR DESCRIPTION
**Pull request by Boppaiah-Acceleon**

### CODE REVIEW

File: README.md
Comment (armandjcovaler) [inline README.md:1]: Please add a line break between line 1 and 2

File: README.md
Comment (armandjcovaler) [inline README.md]: May I request that this be replaced with ## PR Rules to create a subsection

**Comment by armandjcovaler** [comment]
great PR, added some code comments that need's to be resolved before approval to give an idea of what its going to be like working on the acceleon-product-locate repo. in these cases, once you've made those changes just comment on them. in this case the instructions are clear and straight forward so you may resolve the comments once changes have been applied. if you were uncertain or disagree with the request then you'd add comments and the conversation can take place until a resolution is reached.

File: README.md
Comment (Boppaiah-Acceleon) [reply README.md:1]: Added a line break between one and two.

File: README.md
Comment (Boppaiah-Acceleon) [reply README.md]: Added ## to create the sub section.

**Comment by armandjcovaler** [comment]
LGTM (looks good to me) :D and that's as complex as what PRs can get. Will approve and then you can do a rebase and merge

**Review by armandjcovaler — APPROVED** [review]

**Pull request merged**
